methods_and_recursion/mad_libs.rb

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env ruby
+
+def give_me(request)
+  print "#{request} "
+  gets.chomp
+end
+
+adj1 = give_me('Give me an adjective: ')
```

Comment:
You're having to specify `Give me` in multiple lines here, maybe that should go into the `give_me` method?  i.e. `adj1 = give_me('an adjective')`